member/src/main/proto/com/improving/app/member/domain/improving-app-member-domain.proto

```diff
@@ -0,0 +1,198 @@
+syntax =  "proto3";
+
+package com.improving.app.member.domain;
+
+import "com/improving/app/organization/api/improving-app-organization-api.proto";
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  no_default_values_in_constructor: true
+  preserve_unknown_fields: false
+};
+
+
+//Data Model
+message MemberId {
+  option (scalapb.message).extends = "com.improving.app.member.utils.serialize.CborSerializable";
+  string id = 1;
+}
+
+enum MemberType {
+  option (scalapb.enum_options).extends = "com.improving.app.member.utils.serialize.CborSerializable";
+  MEMBER_TYPE_STUDENT = 0;
+  MEMBER_TYPE_FACULTY_STAFF = 1;
+  MEMBER_TYPE_VENDOR = 2;
+  MEMBER_TYPE_SPONSOR =3;
+  MEMBER_TYPE_GENERAL = 4;
+  MEMBER_TYPE_ALUMNI = 5;
+  MEMBER_TYPE_PARENT = 6;
+  MEMBER_TYPE_FAN = 7;
+}
```

Comment:
THe design for this has changed. We don't want to box a member in to be of a type of school person. Instead, they are a member of an organization and the organization can design the "types" of members (roles) which is also tied to the privileges members with that role have. So this enum is not useful.  Remember that the kalix code is only advisory, always compare against he current RIDDL.

Reply:
Removed types from MemberInfo  - will validate against RIDDL